results-out.txt

```diff
@@ -0,0 +1,21 @@
+writing output to results/sparse-full.png
```

Comment:
please remove this file

Reply:
It is already deleted on my repo this line is written on plotting right?

Reply:
I meant delete `results-out.txt`

Reply:
oh okay sorry for the inconvenience